packages/analytics/tsconfig.build.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "./tsconfig.json",
```

Comment:
Considering we have some `tsconfig.*.json` files, it could be nice to have a base file, something like `tsconfig.base.json` just to be more readable what is the basis for every configuration.

Reply:
Having a tsconfig.json was intentional. This is because when you edit code in Visual Studio Code, it will detect files named only as tsconfig.json (AFAIK) and will correctly load the configurations so that tsserver can emit errors/warnings for your code when you are editing it. 

That repository you mentioned I believe is for another purpose. In this case, we need to have a tsconfig.json here not only because of the editing experience but also because of generating the correct output when building.

What do you think?

Reply:
More info about this: https://github.com/microsoft/vscode/issues/12463

Reply:
It's just a suggestion, I'm not a VSCode user, and for me (and maybe other people) it doesn't give any benefits.

IMHO it doesn't hurt to keep this filename, and it probably can be configured in the VSCode to use other files as the basis.

Reply:
Unfortunately tsserver does not allow that configuration ([microsoft/vscode#12463](microsoft/vscode#12463)), we have to use tsconfig.json for VSCode to detect it. (Btw tsserver is the typescript language server that the IDEs can use to add support for typescript in editors, so I believe this might affect any Editor that uses tsserver underneath to provide support for typescript).